script/Cargo.toml

```diff
@@ -22,7 +22,7 @@ ckb-traits = { path = "../traits", version = "= 0.111.0-pre" }
 byteorder = "1.3.1"
 ckb-types = { path = "../util/types", version = "= 0.111.0-pre" }
 ckb-hash = { path = "../util/hash", version = "= 0.111.0-pre" }
-ckb-vm = { version = "=0.24.0", default-features = false }
+ckb-vm = { version = "=0.24.1", default-features = false }
```

Comment:
Please update the changelog in the PR description.

Reply:
done